management/nodes/startupScripts/win2016-201700802-internal.ps1

```diff
@@ -0,0 +1,64 @@
+Set-ExecutionPolicy Unrestricted
+
+$machineSetupLog = 'C:\setup.log'
+$jenkinsServerUrl = $args[0]
+echo ('Jenkins Server URL: {0}' -f $jenkinsServerUrl) | Out-File -Append $machineSetupLog
+$vmName = $args[1]
+echo ('Virtual Machine Name: {0}' -f $vmName) | Out-File -Append $machineSetupLog
+$secret = $args[2]
+echo ('Secret: {0}' -f $secret) | Out-File -Append $machineSetupLog
+
+$winLogonRegistryPath = 'hklm:\software\microsoft\windows nt\currentversion\winlogon\'
+$jenkinsDirectory = 'C:\Jenkins'
+$jenkinsLaunchCmd = Join-Path -Path $jenkinsDirectory -ChildPath 'launch.cmd'
+$jenkinsStartupScript = Join-Path -Path $jenkinsDirectory -ChildPath 'jenkins-windows-startup.ps1'
+$jenkinsLogFile = Join-Path -Path $jenkinsDirectory -ChildPath 'log.txt'
+# The image needs to have LUA off to run properly. If you want the VM to have administrative permissions for
+# everything set $enableLUA to $false.
+$enableLUA = $true
+$enableLUARegistryPath = 'hklm:\software\Microsoft\Windows\CurrentVersion\policies\system\'
+
+# Create the Jenkins directory
+if (!(Test-Path -Path $jenkinsDirectory))
+{
+    Write-Output ('Creating Jenkins Directory: {0}' -f $jenkinsDirectory) | Out-File -Append $machineSetupLog
```

Comment:
>'Creating Jenkins Directory: {0}' -f $jenkinsDirectory [](start = 18, length = 54)

You don't need the `-f` format thing here. You can just interpolate the value in the string using double quotes.